lightkurve/prf/scenemodels.py

```diff
@@ -0,0 +1,670 @@
+"""Provides tools to model a Kepler image."""
+from __future__ import division, print_function
+
+import logging
+from matplotlib import pyplot as plt
+import numpy as np
+from tqdm import tqdm
+
+from oktopus import Prior, GaussianPrior, UniformPrior, PoissonPosterior
+
+from .prfmodels import KeplerPRF
+from ..utils import plot_image
+
+
+__all__ = ['GaussianPrior', 'UniformPrior', 'FixedValuePrior',
+           'StarPrior', 'BackgroundPrior', 'FocusPrior', 'MotionPrior',
+           'StarParameters', 'BackgroundParameters', 'FocusParameters',
+           'MotionParameters', 'SceneModelParameters',
+           'SceneModel', 'PRFPhotometry']
+
+
+log = logging.getLogger(__name__)
+
+
+class FixedValuePrior(Prior):
+    """An improper prior with a likelihood probability of 1 at a single fixed
+    value and -inf elsewhere. This is similar to a Dirac Delta function,
```

Comment:
Since at the top we mention ``probability``, I would replace ``-inf`` with ``0``. Or you can put ``log probability`` at the top and change ``likelihood`` to ``log``.